dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.32/V2_32_14__Update_relationshiptype_bidirectional.sql

```diff
@@ -0,0 +1,15 @@
+alter table relationshiptype
+  add column bidirectional boolean;
```

Comment:
Would be good to make the entire script idempotent by adding "add column if not exists"

Reply:
Will do